app/routes.js

```diff
@@ -0,0 +1,31 @@
+var fs = require('fs');
+	express = require('express');
```

Comment:
sorry about this, wrote this in a hurry forgot to remove the semi-colons. I'm gonna redefine them separately anyways